code/modules/power/generators/turbine.dm

```diff
@@ -255,7 +307,7 @@
 	// This is the power generation function. If anything is needed it's good to plot it in EXCEL before modifying
 	// the TURBGENQ and TURBGENG values
 
-	lastgen = ((compressor.rpm / TURBGENQ)**TURBGENG) * TURBGENQ * productivity
+	lastgen = ((compressor.rpm / TURBGENQ)**TURBGENG) * TURBGENQ * power_curve_mod * productivity
```

Comment:
This will produce about 1,075,174 watts when at 50k RPM and maximum productivity, right before overdrive mode. Just making sure this is the wattage you want. A graph of it can be viewed here: https://www.desmos.com/calculator/6jkfe7pcam

Reply:
Functioning as intended!